book/src/feature/lang_dynamic.md

```diff
@@ -0,0 +1,43 @@
+# Dynamic Types
+
+Darts `dynamic` type is a special type that can hold any type of value. It is
+not per se supported by `flutter_rust_bridge`, but most use cases can likely be
+solved by using an `enum` instead that has a variant for each type you want to
+support.
+
+## Example
+
+Let's say you have a struct that can hold either a `u32` or a `String` and some
+other fields:
+
+```rust,noplayground
+struct MyStruct {
+    a: Optional<u32>,
+    b: Optional<String>,
+}
```

Comment:
well, this does not look like a reasonable usage

Reply:
I was thinking about that as well, but isn't this the alternate (worse) design?

Reply:
Ok, then just say this is a worse design ;)